rocky/katalogus/views/plugin_detail.py

```diff
@@ -84,15 +84,17 @@ def post(self, request, *args, **kwargs):
     def handle_page_action(self, action: str) -> None:
         if action == PageActions.RESCHEDULE_TASK.value:
             task_id = self.request.POST.get("task_id")
-            task = client.get_task_details(task_id)
+            task = client.get_task_details(self.organization.code, task_id)
+            if task:
```

Comment:
Since `get_task_details` returns `Optional[Task]`, you should explicitly check for `None`.
`if task is not None: ...`